src/remembered-redis.ts

```diff
@@ -177,7 +191,15 @@ export class RememberedRedis extends Remembered {
 			const realTtl = ttl || 1;
 			const resultCopy: T = clone(result);
 			if (this.alternativePersistence) {
-				if (!this.waitSaving) {
+				if (!this.alternativePersistence.maxSavingDelay) {
+					await this.persistKeys(
+						{
+							fulfilled: false,
+							entries: new Map([[redisKey, [realTtl, resultCopy]]]),
+						},
+						redisKey.replace(/:/g, '/'),
+					);
```

Comment:
This points allow us to keep the s3 file with the same name of the redis key if there is no maxSavingDelay, for a better refarencing